source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -117,6 +117,105 @@ For a more data-efficient browser preload, we also recommend that you define the
 />
 ```
 
+## Image Sizes
+
+First be sure you have a good understanding of how the browser selects what image to load depending on the image rendered width and the image widths available to it in the srcset attribute ([this](https://ericportis.com/posts/2014/srcset-sizes/#part-2) link is a good start).
```

Comment:
We tend to use `<blockquote>` for these kind of hints. Here is how it could look:

```suggestion
<blockquote class="tip">
**Prerequisites**: be sure you have a good understanding of how the browser selects what image to load. It is a process that depends on the rendered image width and the image widths available in its `srcset` attribute (the [srcset + sizes = AWESOME!](https://ericportis.com/posts/2014/srcset-sizes/#part-2) article is a good explanation).
</blockquote>
```